src/screens/bottomTabs/Explore/Explore.tsx

```diff
@@ -1,7 +1,112 @@
+import React, { useState, useEffect, useCallback, useMemo } from 'react';
+import { FlatList, StyleSheet, RefreshControl } from 'react-native';
+import { useSelector } from 'react-redux';
+import { Card } from 'src/components';
 import { ScreenTemplate } from 'src/components/templates';
+import { useAppDispatch } from 'src/store';
+import {
+  getAccommodationList,
+  getAccommodationListLoading,
+  getFilterSettings,
+} from 'src/store/selectors';
+import { AsyncThunks } from 'src/store/thunks';
+import { ExploreListItem } from 'src/types';
+import { getAccommodationsListQuery } from 'src/utils';
+
+interface CardProps extends ExploreListItem {}
 
 const Explore = () => {
-  return <ScreenTemplate></ScreenTemplate>;
+  const dispatch = useAppDispatch();
+  const filter = useSelector(getFilterSettings);
+  const data = useSelector(getAccommodationList);
+  const loading = useSelector(getAccommodationListLoading);
+  const [pageNumber, setPageNumber] = useState<number>(1);
+  const [isRefreshing, setIsRefreshing] = useState<boolean>(false);
+
+  const renderItem = useMemo(
+    () =>
+      ({ item }: { item: CardProps }) => (
+        <Card
+          id={item.id}
+          thumbnailUrl={item.thumbnailUrl}
+          squareMeters={item.squareMeters}
+          numberOfRooms={item.numberOfRooms}
+          allowedNumberOfPeople={item.allowedNumberOfPeople}
+          price={item.price}
+          address={item.address}
+        />
+      ),
+    []
+  );
+
+  const fetchData = useCallback(async () => {
+    try {
```

Comment:
remove try-catch block, we already have it on Thunk

Reply:
removed